### PR TITLE
KevS-SK1298-Update-Create-Shared-SQL-for-WebHosting

### PR DESCRIPTION
**Pull request by Kazarus4**
Updates : 

- Date of last update 
- add paragraph on step 1 about the new "status" of shared SQL
- completed the second prerequisite
-  Update the first screen to shaw the new column in the db dashboard

### CODE REVIEW

**Comment by Kazarus4** [comment]
need peer rv by @Y0Coss for FRFR

File: pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md [hunk 1]
```diff
@@ -26,6 +26,18 @@ Le tableau de cette section contient toutes les bases de données créées avec
 
 ![databasecreation](images/database-creation-step1.png){.thumbnail}
 
+Dans le tableau, les bases de données peuvent disposer de plusieurs « statuts » différents :
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md]:
```suggestion
Dans le tableau, les bases de données peuvent disposer de plusieurs statuts différents :
```

File: pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md [hunk 2]
```diff
@@ -26,6 +26,18 @@ Le tableau de cette section contient toutes les bases de données créées avec
...
+
+- **Incluse** : indique que la base de données est comprise dans votre offre d’hébergement web. Elle n’engendre pas de surcoûts supplémentaires.</br></br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md]:
```suggestion
- **Incluse** : indique que la base de données est comprise dans votre offre d’hébergement web. Elle n’engendre pas de surcoût supplémentaire.</br></br>
```

File: pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md [hunk 3]
```diff
@@ -26,6 +26,18 @@ Le tableau de cette section contient toutes les bases de données créées avec
...
+- **Optionnelle** : indique que la base de données a été souscrite en complément des bases de données incluses avec votre hébergement web. Vous payez un complément pour disposer de cette base de données supplémentaire sur votre hébergement web.</br></br>
+- **Incluse - retirée de la vente** : indique que la base de données incluse va bientôt être retirée de la vente et devenir obsolète. </br>Nous vous recommandons, *avant* que la base de données ne devienne obsolète, de récupérer son contenu pour le placer dans une nouvelle base de données plus récente (dont l'obsolescence n'est pas encore programmée).</br></br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md]:
```suggestion
- **Incluse - retirée de la vente** : indique que la base de données incluse va bientôt être retirée de la vente et devenir obsolète. </br>Nous vous recommandons, **avant** que la base de données ne devienne obsolète, de récupérer son contenu pour le placer dans une nouvelle base de données plus récente (dont l'obsolescence n'est pas encore programmée).</br></br>
```

File: pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md [hunk 4]
```diff
@@ -26,6 +26,18 @@ Le tableau de cette section contient toutes les bases de données créées avec
...
+- **Optionnelle - retirée de la vente** : indique que la base de données souscrite en complément sur votre hébergement web va bientôt être retirée de la vente et devenir obsolète. </br>Nous vous recommandons, *avant* que la base de données ne devienne obsolète, de récupérer son contenu pour le placer dans une nouvelle base de données plus récente (dont l'obsolescence n'est pas encore programmée).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md]:
```suggestion
- **Optionnelle - retirée de la vente** : indique que la base de données souscrite en complément sur votre hébergement web va bientôt être retirée de la vente et devenir obsolète. </br>Nous vous recommandons, **avant** que la base de données ne devienne obsolète, de récupérer son contenu pour le placer dans une nouvelle base de données plus récente (dont l'obsolescence n'est pas encore programmée).
```

File: pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md [hunk 5]
```diff
@@ -26,6 +26,18 @@ Le tableau de cette section contient toutes les bases de données créées avec
 
 ![databasecreation](images/database-creation-step1.png){.thumbnail}
 
+Dans le tableau, les bases de données peuvent disposer de plusieurs statuts différents :
+
+- **Incluse** : indique que la base de données est comprise dans votre offre d’hébergement web. Elle n’engendre pas de surcoût supplémentaire.</br></br>
+- **Optionnelle** : indique que la base de données a été souscrite en complément des bases de données incluses avec votre hébergement web. Vous payez un complément pour disposer de cette base de données supplémentaire sur votre hébergement web.</br></br>
+- **Incluse - retirée de la vente** : indique que la base de données incluse va bientôt être retirée de la vente et devenir obsolète. </br>Nous vous recommandons, **avant** que la base de données ne devienne obsolète, de récupérer son contenu pour le placer dans une nouvelle base de données plus récente (dont l'obsolescence n'est pas encore programmée).</br></br>
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md]:
```suggestion
- **Incluse - retirée de la vente** : indique que la base de données incluse va bientôt être retirée de la vente et devenir obsolète. </br>Nous vous recommandons, **avant** que la base de données ne devienne obsolète, de récupérer son contenu pour le placer dans une nouvelle base de données plus récente (dont la fin de vente n'est pas encore programmée).</br></br>
```

File: pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md [hunk 6]
```diff
@@ -26,6 +26,18 @@ Le tableau de cette section contient toutes les bases de données créées avec
...
+- **Optionnelle - retirée de la vente** : indique que la base de données souscrite en complément sur votre hébergement web va bientôt être retirée de la vente et devenir obsolète. </br>Nous vous recommandons, **avant** que la base de données ne devienne obsolète, de récupérer son contenu pour le placer dans une nouvelle base de données plus récente (dont l'obsolescence n'est pas encore programmée).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/sql_create_database/guide.fr-fr.md]:
```suggestion
- **Optionnelle - retirée de la vente** : indique que la base de données souscrite en complément sur votre hébergement web va bientôt être retirée de la vente et devenir obsolète. </br>Nous vous recommandons, **avant** que la base de données ne devienne obsolète, de récupérer son contenu pour le placer dans une nouvelle base de données plus récente (dont la fin de vente n'est pas encore programmée).
```

**Pull request merged**